### ISSUE TITLE
options backup pool and DNS refresh

### ISSUE THREAD

**Issue by M5M400** [fireice-uk/xmr-stak-cpu#151]
Hi,

I've had some problems with supportXMR.com pool lately, as the new server I added kept going dark. It would have been tremendously useful for the miners if stak products would

a) allow for the configuration of a backup pool server (for obvious reasons)
b) ditch cached DNS / re-resolve DNS before a reconnect attempt (my DNS TTL is set to 5 minutes and I set it back to the IP of the old server, but miners keep connecting to the crashed box)

**Comment by fireice-uk** [comment]
@M5M400 you are correct, the miner will resolve the DNS only once (if successful) and store the result. Restarting the miner is a quick fix for that.

Nice solution would be to flush the cached result every second failure.

**Comment by fireice-uk** [comment]
@M5M400 I actually found was talking out of my tailpipe here when I read the code. Are you sure that wasn't the local OS cache?

**Comment by M5M400** [comment]
Not entirely sure and perfectly possible. Let's close that one.

**Comment by M5M400** [comment]
Or leave it open for the "backup pool" feature ;)

**Comment by M5M400** [comment]
being stupid

**Comment by BigslimVdub** [comment]
any update on backup pool? I would like using my smaller pool but with all these DDos attacks going around smaller pools I almost have to stick to nano pool. SGminer supports backup pool along with Claymore.

**Comment by SuperBartimus** [comment]
I'm really liking the idea of a backup pool option too
